jest.config.js

```diff
@@ -6,14 +6,29 @@ module.exports = {
   roots: ['<rootDir>/src'],
   collectCoverageFrom: ['src/**/*.{js,jsx}'],
   coveragePathIgnorePatterns: ['__tests__', 'index'],
+
   // Test environment that simulates a browser (using jsdom)
   testEnvironment: 'jsdom',
   testPathIgnorePatterns: ['/node_modules/', '<rootDir>/src/__tests__/mocks/'],
 
   // Jest transformations -- this adds support for TypeScript and ES6+ syntax
   transform: {
-    '^.+\\.jsx?$': 'babel-jest', // Use babel-jest for JavaScript files
+    '^.+\\.(j)sx?$': [
+      '@swc/jest',
+      {
+        jsc: {
+          transform: {
+            react: {
+              runtime: 'automatic',
+            },
+          },
+        },
+      },
+    ],
   },
+  transformIgnorePatterns: [
+    '/node_modules/(?!(taskcluster-client-web|data-uri-to-buffer|fetch-blob|formdata-polyfill|cross-fetch)/)',
```

Comment:
IIRC these ones were used in perfcompare especially because of the taskcluster client, it may not be needed here (or maybe not for all these entries?)

Reply:
Yes, I was a bit tired when I did this so thanks for catching this. Some cleanup is definitely needed.